src/test/java/org/folio/rest/impl/CashDrawerReconciliationReportTest.java

```diff
@@ -208,6 +215,35 @@ public void validReportWhenPaymentsExist() {
           buildReportTotalsEntry(FEE_FINE_OWNER_TOTALS, "16.00", "4")))));
   }
 
+  @Test
+  public void sources() {
```

Comment:
Add more tests - 1) sources for a fee/fine action with a different service point ID